src/grid/rtransform.py

```diff
@@ -24,14 +24,19 @@
 from abc import ABC, abstractmethod
 from numbers import Number
 
-from grid.basegrid import OneDGrid, RadialGrid
+from grid.basegrid import OneDGrid
 
 import numpy as np
 
 
 class BaseTransform(ABC):
     """Abstract class for transformation."""
 
+    @property
+    def domain(self):
+        """(int, int): the range of grid points."""
+        return self._domain
+
```

Comment:
I would assume the domain depends on the input of the transform. Can we fix it like this?

Reply:
You can apply the `transform` method to the input `oned_grid.domain` to get the domain of the resulting grid.

Reply:
done.